### DISCUSSION TITLE
Tag people in discussions

### DISCUSSION THREAD

**Discussion by lexoyo** [Leantime/leantime#1202] [Ideas]
Hello

At internet2000.net we are missing a way to tag / ping people in the discussions, so that they get notified of an answer or question, even if the card is not theirs

I'm talking about something like this:
![aid8803912-v4-728px-Tag-Someone-on-WhatsApp-on-Android-Step-7](https://user-images.githubusercontent.com/715377/207293164-5149c9aa-dc0c-4ccd-b98b-8692f170dae5.jpg)

Is there a way to do this? A plugin? Or an easy way to code the feature as a contribution?

**Comment by lexoyo** [comment]
I'm moving this to "ideas" since it is probably more of a feature request

The need is to notify users from specific comments/discussions

**Comment by lexoyo** [comment]
I ended up creating a bot with n8n.io
It get notifications from the slack web hook and when it finds a `@{ name }` it sends an email to `{ name }@mycompany.com

**Comment by marcelfolaron** [comment]
This is implemented now!
